allensdk/brain_observatory/behavior/data_objects/stimuli/presentations.py

```diff
@@ -599,10 +599,12 @@ def _get_spontaneous_stimulus(
                 }
             )
         # Check for 5 minute gray screen stimulus block at the start of the
-        # movie
+        # movie. We give some leeway around 5 minutes at 285 seconds to account
+        # for some sessions which have start times slightly less than 300
+        # seconds.
         if (
             stimulus_presentations_table.iloc[0]["start_frame"] > 0
-            and stimulus_presentations_table.iloc[0]["start_time"] > 0
+            and stimulus_presentations_table.iloc[0]["start_time"] > 285
```

Comment:
This block would currently still run if `start_time` is anything > 0, including if it's 290 for example, so I don't understand the need for this change. We're not checking for start_time == 300

Reply:
There can be a short delay that is not the "5 minutes grey screen" and, according to Marina, we don't want to add those to the set of stimulus blocks.